src/Fixer/Alias/EregToPregFixer.php

```diff
@@ -49,7 +49,7 @@ public function getDefinition()
     {
         return new FixerDefinition(
             'Replace deprecated `ereg` regular expression functions with preg.',
-            [new CodeSample('<?php $x = ereg(\'[A-Z]\');')],
+            [new CodeSample("<?php \$x = ereg('[A-Z]');\n")],
```

Comment:
why are all those new lines needed?

Reply:
I was wondering too. I guess this is required to avoid seeing a bunch of `\ No newline at end of file` in code samples.

Reply:
@julienfalque is right here, the diff samples do not look good without the ending line break, still correct unified diff but with the `\No newline at end of file` notice